doc/api/schemas/CodeRepositorySettings.yaml

```diff
@@ -0,0 +1,40 @@
+type: object
+description: Code Repository Settings
+properties:
+    projectUrl:
+        type: string
+    environment:
+        type: string
+    deploymentModes:
+        type: array
+        items:
+            $ref: './DeploymentMode.yaml'
+    entryPoints:
+        type: array
+        items:
+            $ref: './EntryPoint.yaml'
```

Comment:
`entryPoints` is completely inherited from the `mlproject` specification of mlflow. If we define entry points on the CodeRepo level, we implicitly make strong assumptions on the structure of said repo.